e2e/package.json

```diff
@@ -7,11 +7,14 @@
     "cypress-file-upload": "3.1.2",
     "express": "4.17.1",
     "merge-deep": "3.0.2",
-    "mocha-junit-reporter": "1.22.0"
+    "mocha-junit-reporter": "1.22.0",
+    "start-server-and-test": "1.9.1"
   },
   "scripts": {
     "cypress:run": "cypress run",
     "cypress:open": "cypress open",
-    "start:webhook": "node webhook_serve.js"
+    "start:webhook": "node webhook_serve.js",
+    "test": "start-test start:webhook 3000 test:group",
+    "test:group": "ls -d cypress/integration/** | xargs -t -I % sh -c '{ npx cypress run --spec=\"%/**/*\"; }'"
```

Comment:
Once the enterprise directory makes it's way into master, this will run it. Initially this will be a problem since we don't have ES set up on the cypress server. Not sure if we want to take that into account now or later.

Reply:
Since we'll run all on Cypress server, I think that's fine to fail at first but let's aim to have ES installed on as much as possible 😝